first-contract/README.md

```diff
@@ -0,0 +1,190 @@
+# Writing your first contract on Optimism
+
+[![Discord](https://img.shields.io/discord/667044843901681675.svg?color=768AD4&label=discord&logo=https%3A%2F%2Fdiscordapp.com%2Fassets%2F8c9701b98ad4372b58f13fd9f65f966e.svg)](https://discord.com/channels/667044843901681675)
+[![Twitter Follow](https://img.shields.io/twitter/follow/optimismPBC.svg?label=optimismPBC&style=social)](https://twitter.com/optimismPBC)
+
+This tutorial assumes that you know how to program, but that you've never dealt with smart contracts, much less written one.
+If you are already familiar with Solidity [go here](../getting-started) for a tutorial that teaches you only how to use Optimism with the tools you already know.
+
+## Setup 
+
+### Wallet
+
+Before you can use Ethereum you need [wallet software](https://cointelegraph.com/news/overview-of-software-wallets-the-easy-way-to-store-crypto).
+In this tutorial we use [MetaMask](https://metamask.io/), a browser extension, as our wallet.
+You can also use MetaMask in a mobile device (iOS or Android), but for software development it is best to use a notebook or desktop computer.
+
+1. Make sure you have either Chrome, Firefox, Brave, or Edge. 
+
+1. Browse to [MetaMask](https://metamask.io/), scroll down and choose to install MetaMask for your browser.
+
+1. Click **Get Started**.
+
+1. Click **Create a wallet** and then **I agree**.
+
+1. Type a password, accept the terms, and click **Create**.
+
+1. Click **Next**, reveal the secret words, and write them down somewhere safe. 
+   We plan to only use this wallet for testing, so the key phrase should not matter that much, but it's best to develop good habits from the beginning.
+
+1. Let MetaMask help you confirm you have the correct written key phrase.
+
+1. Copy your address, you'll need it soon.
+
+### Blockchain
+
+In this tutorial we'll use [Optimism](https://www.optimism.io/), a rollup network, to run our contracts.
+To avoid spending real money, we'll use Optimistic Kovan, a test network built on top of Kovan.
+
+1. [Go here](https://chainid.link/?network=optimism-kovan) to add Optimistic Kovan to your wallet. 
+Click **Connect** on the web page and then **Approve** in MetaMask.
+
+1. Permit the network switch.
+
+1. Go to [this faucet](https://faucet.paradigm.xyz/) to get ETH on the Optimistic Kovan network.
+
+1. Paste your address, select **Drip on additional network**, and click **Claim**.
+
+1. [Go to Etherscan](https://kovan-optimistic.etherscan.io/), search for your address, and see that you now have 1 ETH.
+
+
+## Interacting with a Solidity contract
+
+The most common smart contract development language is called [Solidity](https://docs.soliditylang.org/en/v0.8.11/).
+Here we will use using an integrated development environment (IDE) called [Remix](https://remix.ethereum.org/).
```

Comment:
Why? We will use Remix for this tutorial as it requires the least set up (it runs in your browser)